packages/execution-environments/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
+
+const DIST = path.resolve(__dirname, 'dist');
+
+module.exports = (_, argv) => {
+  const isProd = argv.mode === 'production';
+
+  let extraOptions = {};
+
+  if (isProd === false) {
+    extraOptions = {
+      devtool: 'inline-source-map',
+    };
+  }
+  const config = {
+    ...extraOptions,
+    mode: argv.mode,
+    entry: {
+      iframe: './src/iframe/index.ts',
+      webworker: './src/web-workers/index.ts',
+    },
+    output: {
+      filename: '[name].bundle.js',
+      path: DIST,
+    },
+    plugins: [new NodePolyfillPlugin()],
```

Comment:
We should ensure that SES lockdown is copied purely as in [the orignal implementation](https://github.com/MetaMask/iframe-execution-environment/blob/main/webpack.config.js#L35-L42), and added to `index.html` via its own `<script>` tag, which should also be the first script to execute.